### PR TITLE
Add function to run_occlum_passed_test.sh

### PR DESCRIPTION
**Pull request by daan3601**

### CODE REVIEW

File: occlum/run_occlum_passed_tests.sh [hunk 1]
```diff
@@ -1,54 +1,47 @@
 #!/bin/bash
 
 set -x
-
 TEST_BIN_DIR=/opt/occlum/gvisor_syscall_tests
-BLOCKLIST_DIR=blocklist/
-TEST_LIST=occlum_test_list.txt
+BLOCKLIST_DIR=/blocklist/
```
Comment (jessehui) [inline occlum/run_occlum_passed_tests.sh]: This is absolute address still. Don't use absolute address.

File: occlum/run_occlum_passed_tests.sh [hunk 2]
```diff
@@ -1,54 +1,47 @@
...
-        ret=$?
-        popd
-        rm -rf occlum_workspace
-    else
-        echo "Warning: test does not exist"
-        ret=1
-    fi
-
-    return $ret
+run_one_test(){
```
Comment (jessehui) [inline occlum/run_occlum_passed_tests.sh:24]: This function runs all the tests. This name is not suitable anymore.

File: occlum/run_occlum_passed_tests.sh [hunk 3]
```diff
@@ -1,54 +1,47 @@
...
 RESULT=0
 FILTER=""
 
-get_blocklisted_subtests() {
-	if [ -f $BLOCKLIST_DIR/$1 ]; then
-        FILTER=$(cat $BLOCKLIST_DIR/$1 | tr '\n' ':')
-        return 0
+get_blocklist_subtests(){
+	if [ -f $BLOCKLIST_DIR/$1 ];then
+	FILTER=$(cat $BLOCKLIST_DIR/$1 | tr '\n' ':')
```
Comment (jessehui) [inline occlum/run_occlum_passed_tests.sh]: The indent is not good.

**Review by jessehui — CHANGES_REQUESTED** [review]

**Comment by jessehui** [comment]
TEST

File: occlum/run_occlum_passed_tests.sh [hunk 4]
```diff
@@ -1,60 +1,65 @@
...
+		occlum exec /bin/$1 --gtest_filter=-$FILTER
+		ret=$?
+	else
+		echo "Warning: test does not exit"
+		ret=1
+	fi
+	return $ret
 }
 
+occlum new occlum_workspace
```
Comment (jessehui) [inline occlum/run_occlum_passed_tests.sh:38]: What if there exists the occlum_workspace? Maybe delete the old occlum_workspace before creating new workspace.

File: occlum/run_occlum_passed_tests.sh [hunk 5]
```diff
@@ -1,60 +1,65 @@
...
 
+occlum stop
 printf "$RED$PASSED_TESTS$NC of $RED$TESTS$NC test suites are passed\n"
-
 [ $PASSED_TESTS -ne $TESTS ] && RESULT=1
+printf "The $(($TESTS-$PASSED_TESTS)) failed test suites in this run are as follows:\n"
+cat log
+rm log
+popd
+rm -rf occlum_workspace
```
Comment (jessehui) [inline occlum/run_occlum_passed_tests.sh]: Don't delete the workspace. This may needs inspection after finishing.

**Review by jessehui — APPROVED** [review]
LGTM

**Review by guzongmin — APPROVED** [review]
LGTM

**Pull request merged**